non-critical-infra/packages/hydra-queue-runner/default.nix

```diff
@@ -0,0 +1,104 @@
+{
+  rustPackages,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  zlib,
+  protobuf,
+  lib,
+  makeWrapper,
+  nixVersions,
+  nlohmann_json,
+  libsodium,
+  boost,
+}:
+let
+  version = "unstable-2025-08-07";
+  src = fetchFromGitHub {
+    owner = "helsinki-systems";
+    repo = "hydra-queue-runner";
+    rev = "06301cf6b070c016d749bc71cebffe2b82a0e3ac";
+    hash = "sha256-NiI1pHAmiCfETZN2IEg0FgUrARXpmCcTSZwk/ymnvU8=";
+  };
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-oNUMmFfts4rjBX0k5mzsxpYA2JqgsRu1nMRFf/2rZa8=";
+  nativeBuildInputs = [
+    pkg-config
+    protobuf
+    makeWrapper
+  ];
+  buildInputs = [
+    openssl
+    zlib
+    protobuf
+
+    nixVersions.nix_2_29
```

Comment:
also builds and works with 2_30, but i think you are currently still using 2_29 ?